.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,97 @@
+---
+name: Bug Report
+description: Create a report to help us improve.
+labels: [bug]
+assignees: aio-libs/triagers
+body:
+- type: markdown
+  attributes:
+    value: |
+      **Thanks for taking a minute to file a bug report!**
+
+      ⚠
+      Verify first that your issue is not [already reported on
+      GitHub][issue search].
+
+      _Please fill out the form below with as many precise
+      details as possible._
+
+      [issue search]: ../search?q=is%3Aissue&type=issues
+
+- type: textarea
+  attributes:
+    label: Describe the bug
+    description: >-
+      A clear and concise description of what the bug is.
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: To Reproduce
+    description: >-
+      Describe the steps to reproduce this bug.
+    placeholder: |
+      1. Have certain environment
+      2. Run given code snippet in a certain way
+      3. See some behavior described
```

Comment:
Had to re-read this a couple of times. "Have certain environment" could be rewritten to "Your environment (OS, etc.)". But yea please expand on this; could be confusing at first.

Reply:
I agree, I've changed the placeholder to:
```
1. Create a certain environment (OS, Environment Variables, etc)
2. Run a code snippet '...'
3. An error occurs
```
is that clear enough or should I think of another placeholder?

Reply:
Maybe 3. Describe the error that is occurring